Brand-Guide.md

```diff
@@ -32,22 +32,6 @@ These components will collectively create a professional and scientific visual i
 
 A strong and consistent brand design will enhance BIDS' recognition within the neuroscience community and effectively communicate its values and goals, fostering greater trust and engagement.
 
-## About Brain Imaging Data Structure (BIDS)
-
-BIDS is a community-driven specification standard designed to restructure the organization and sharing of neuroimaging and behavioral data. Historically, the lack of a standardized format for neuroimaging data led to inconsistencies and inefficiencies, even among researchers within the same lab. BIDS therefore addresses this issue by providing a simple, intuitive, and easily adaptable framework that ensures data consistency and facilitates collaboration.
-
-Inspired by the internal format of the OpenNeuro repository, BIDS was developed with contributions from a diverse group of neuroscientists. The specification is based on straightforward file formats and folder structures that reflect current lab practices, making it accessible to scientists from various backgrounds.
-
-BIDS has fostered a robust ecosystem of tools and resources, including:
-
--	The BIDS specification, detailing data organization.
--	The BIDS starter kit, offers a simple introduction.
--	The BIDS validator, an automated tool for checking dataset compliance.
--	BIDS Apps, a collection of neuroimaging pipelines compatible with BIDS datasets.
--	OpenNeuro, a repository for BIDS-formatted datasets.
-
-This ecosystem continues to grow, ensuring that BIDS remains the gold standard for neuroimaging data organization.
-
```

Comment:
This section is unnecessary as any user can go find this same information on the BIDS Website or the GitHub "awesome-bids" repository. Let me know if you feel differently, of course.

Reply:
That is true